Package.swift

```diff
@@ -60,3 +61,25 @@ let package = Package(
         ),
     ]
 )
+
+func loadSwiftLintVersion() -> Version {
+    guard let yamlString = try? String(contentsOf: URL(fileURLWithPath: #file)
+                                        .deletingLastPathComponent()
+                                        .appendingPathComponent(".swiftlint.yml")) else {
+        fatalError("Failed to read YAML file.")
+    }
+
+    guard let versionLine = yamlString.components(separatedBy: .newlines)
+            .first(where: { $0.contains("swiftlint_version") }) else {
+        fatalError("SwiftLint version not found in YAML file.")
+    }
+
+    // assumes the format `swiftlint_version: <version>`
+    guard let version = Version(versionLine.components(separatedBy: ":")
+                                    .last?
+                                    .trimmingCharacters(in: .whitespaces) ?? "") else {
+        fatalError("Failed to extract SwiftLint version.")
+    }
+
+    return version
+}
```

Comment:
Love it. 

It's a bummer we'll have to add it to every `Package.swift`. But, any centralization attempt would be vain because we'd need a different dependency manager to fetch it ♻️ .

One question, I'm not sure what the rule behind the indentation is.

I run it in Xcode and got:

```suggestion
func loadSwiftLintVersion() -> Version {
    guard let yamlString = try? String(contentsOf: URL(fileURLWithPath: #file)
        .deletingLastPathComponent()
        .appendingPathComponent(".swiftlint.yml")) else {
        fatalError("Failed to read YAML file.")
    }

    guard let versionLine = yamlString.components(separatedBy: .newlines)
        .first(where: { $0.contains("swiftlint_version") }) else {
        fatalError("SwiftLint version not found in YAML file.")
    }

    // Assumes the format `swiftlint_version: <version>`
    guard let version = Version(versionLine.components(separatedBy: ":")
        .last?
        .trimmingCharacters(in: .whitespaces) ?? "") else {
        fatalError("Failed to extract SwiftLint version.")
    }

    return version
}
```

I point this out only because if we replicate this code in other projects, I would be good to have it consistent.

Reply:
> But, any centralization attempt would be vain because we'd need a different dependency manager to fetch it ♻️ .

I was wondering if another package acting as a wrapper plugin around SwiftLint and adding this extra functionality would work. Perhaps something to try out in the future.